custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -49,7 +58,18 @@ def aggregation_query(self):
                    awcm.block_name, awcm.supervisor_name, awcm.awc_name, awcm.aww_name,
                    awcm.contact_phone_number, awcm.wer_weighed, awcm.wer_eligible,
                    awcm.awc_days_open
-        )
+        );
+        /* update expected visits for cf cases (not in agg_ccs_record */
+        UPDATE {tablename} perf
+        SET expected_visits = expected_visits + ucr.expected
+        FROM (
+             SELECT SUM(0.39) AS expected, awc_id
+             FROM {ccs_record_case_ucr} 
+             WHERE %(month)s - add > 183 AND closed = 0
```

Comment:
I think that `closed = 0` should be `closed_on IS NULL OR closed_on < %(next_month)s`

`closed` represents the current state of the case, and this is calculating the report based on the previous month's performance. Which would mean if a case was closed during the previous month or in the first 10 days of the current month, it wouldn't be reflected in the previous months calculation.

Reply:
Used a slightly different one to avoid having to calculate next month but I think it should be the same